windows-agent/internal/proservices/proservices_test.go

```diff
@@ -106,6 +110,17 @@ func TestNew(t *testing.T) {
 	}
 }
 
+func checkFileExists(path string) func() bool {
+	return func() bool {
+		return fileExists(path)
+	}
+}
+
+func fileExists(path string) bool {
```

Comment:
This function is only called once, so you can inline it in checkFileExists if it’s not reused :)